doc/genmodel.py

```diff
@@ -149,8 +149,10 @@ def make_figure(model_info, opts):
     """
     import matplotlib.pyplot as plt
 
+    print("Build model")
```

Comment:
There is a number of print statements introduced. As I understand they were added for debugging purposes.  Do we still need them